machine_learning/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(machine_learning)
+
+find_package(catkin REQUIRED)
+
+# catkin_python_setup()
```

Comment:
I wasn't planning on exposing any of the inner workings to the system, therefore it wouldn't need to be there.
The image provider class could be helpful externally, but I don't think the machine learning folder is where it should live. Until there's a nice home for it set up (maybe in `mil_dev_tools`), it should stay as a private file only used by the labeler.